docs/Installation.md

```diff
@@ -38,18 +38,6 @@ This instruction covers the following operating systems:
 
 ### 1. CentOS
 
-Install binary dependencies:
-
-```sh
-sudo yum install perl-JSON-XS
-```
-
-Install dependencies from CPAN:
-
-```sh
-sudo cpanm Locale::Msgfmt Locale::TextDomain MooseX::Getopt Text::Reflow
```

Comment:
If dependencies are explicitly installed from CPAN or indirectly installed when Zonemaster::CLI is installed from CPAN makes no difference. I cannot see the value of keeping this.

Reply:
Maintaining the list of yum packages is significantly easier when we have this list spelled out.

Reply:
Do you mean "searching for RPMs is significantly easier when we have this list spelled out"? -- true.

Reply:
Well, I think the greatest improvement is in knowing what RPMs to search for.

Reply:
True.